lesson_2/Lesson_2.4.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>0.3 -- non-English speaking audiences will understand that better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Thanks </p>